web-app/src/services/http.ts

```diff
@@ -0,0 +1,74 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { get, isNil } from 'lodash';
+import { Notify } from 'quasar';
+import internalApi from './internalApi';
+
+const api = {
+  internalApi,
+};
+
+export default api;
+export const jc = axios.create({
+  headers: {},
+  baseURL: process.env.BACKEND_API_URL,
+});
+
+jc.interceptors.request.use(request => addToken(request));
+
+const addToken = (config: AxiosRequestConfig) => {
+  // TODO: get cookie 2x
+  if (
+    'TokenManager.getAccessToken() - get from cookie' &&
+    config.url !== 'refresh'
+  ) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    config.headers['Authorization'] =
+      'Bearer ' + 'TokenManager.getAccessToken()';
+  }
+  return config;
+};
```

Comment:
What this function actually does is extracting token from cookie and sending it as a Bearer token which is totally WRONG! Cookies are automatically sent alongside request and they DON'T need to (actually it is  MISTAKE)  be converted to Bearer token. It's distracting whole philosophy of using cookies with request. We want cookies so we use cookies, not Bearer tokens!!!